tests/routes/contacts.test.js

```diff
@@ -42,11 +42,15 @@ beforeAll(async () => {
   await Contact.destroy(query);
 });
 
-afterAll(async done => {
-  server.close(done);
-  await Case.destroy(query);
-  await CSAMReport.destroy(query);
-  await Contact.destroy(query);
+afterAll(done => {
+  server.close(() => {
+    Case.destroy(query).then(() => {
```

Comment:
I tried to promisify the server close callback without success (the promise never seemed to resolve) so just decided to do it old school for now. If anyone can neaten it up further, please show me!